openarm_hardware/include/openarm_hardware/openarm_hardware.hpp

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) 2025, Reazon Holdings, Inc.
-// Copyright (c) 2025, Stogl Robotics Consulting UG (haftungsbeschränkt) (template)
+// Copyright (c) 2025, Stogl Robotics Consulting UG (haftungsbeschränkt)
+// (template)
```

Comment:
Could you remove all `(template)` in `Copyright (c) 2025, Stogl Robotics Consulting UG (haftungsbeschränkt) (template)`? It seems that it's garbage.

```suggestion
```

Reply:
Hmm, https://github.com/b-robotized/ros_team_workspace/blob/master/templates/licenses/default_cpp.txt includes `(template)`...

Could you ask https://github.com/b-robotized/ros_team_workspace whether `(template)` is intentional or not?

Reply:
Sure!